src/lib/isIdentityCard.js

```diff
@@ -185,13 +186,8 @@ const validators = {
       const mm = parseInt(birDayCode.substring(4, 6), 10);
       const dd = parseInt(birDayCode.substring(6), 10);
       const xdata = new Date(yyyy, mm - 1, dd);
-      if (xdata > new Date()) {
-        return false;
-        // eslint-disable-next-line max-len
-      } else if ((xdata.getFullYear() === yyyy) && (xdata.getMonth() === mm - 1) && (xdata.getDate() === dd)) {
```

Comment:
Could you explain this rewrite?

Reply:
Or check if you rebased correctly with the latest `master`.

Reply:
xdata is created using `yyyy, mm-1, and dd`. 
Getting `xdata.getFullYear()` and comparing it with what it was used to create it, it will always be `true`. 
So, that line seemed redundant and of no use(it is like creating x from w, which is equal to 1 and comparing x.w with w or 1`.

Reply:
Got it, thanks.